File: tempCodeRunnerFile.cpp
/*
 * @Author: NEFU AB_IN
 * @Date: 2021-07-16 18:21:31
 * @FilePath: \Vscode\test.cpp
 * @LastEditTime: 2021-08-21 19:08:48
 */
#include<bits/stdc++.h>
using namespace std;
#define LL                          long long
#define MP                          make_pair
#define SZ(X)                       ((int)(X).size())
#define IOS                         ios::sync_with_stdio(false);cin.tie(0);cout.tie(0);
#define DEBUG(X)                    cout << #X << ": " << X << endl;
typedef pair<int , int>             PII;

signed main()
{
    IOS;
    int t;
    cin >> t;
    while(t --){
        int n, k;
        cin >> n >> k;
        if(n == 1) {
            cout << "No!\n";
            continue;
        }
        if((n & k)) cout << "Yes!\n";
        else cout << "No!\n";
    }
    return 0;
}

File: ACM/Acwing/tempCodeRunnerFile.py
'''
Author: NEFU AB-IN
Date: 2022-01-16 00:23:00
FilePath: \ACM\Acwing\tempCodeRunnerFile.py
LastEditTime: 2022-04-01 17:42:37
'''


def gcd2(a, b):
    if a < b: a, b = b, a
    return gcd2(b, a - b) if b else a


print(gcd2(6, 10))